astropy/units/core.py

```diff
@@ -113,14 +114,25 @@ class _UnitRegistry(object):
     Manages a registry of the enabled units.
     """
     def __init__(self, init=[], equivalencies=[]):
-        if isinstance(init, _UnitRegistry):
-            equivalencies = init.equivalencies
-            init = init.all_units
-
         self._reset_units()
```

Comment:
That would be brilliant! It seems one can get a backport via https://pypi.python.org/pypi/chainmap/1.0.0 (oddly, it seems to suggest it should exist on python 2.7...

And at least on my machine, I seem to be able to do:
```
from distlib.compat import ChainMap
```
which comes from `python-distlib` (which I think astropy relies on, but perhaps only for packaging).